pkg/controller/controller.go

```diff
@@ -0,0 +1,58 @@
+package controller
```

Comment:
I would rename this file since it has nothing to do with controller right? Perhaps `kubeconfigs.go` or something like that.

Reply:
done